extensions/azurePublishNew/src/node/index.ts

```diff
@@ -29,21 +38,64 @@ const createAzurePublishPlugin = (): PublishPlugin<ProvisionConfig> => {
     );
   };
 
-  const getStatus = async () => {
+  const toPublishResponse = (processStatus: ProcessStatus): PublishResponse => {
+    const { id, status, message, log, comment, time } = processStatus;
+
     return {
-      status: 200,
+      status,
       result: {
-        time: new Date().toString(),
-        message: 'Publish successful.',
-        log: '',
+        comment,
+        id,
+        log: log.map((item) => `---\n${JSON.stringify(item, null, 2)}\n---\n`).join('\n'),
+        message,
+        status,
+        time: time.toString(),
       },
     };
   };
 
+  const getStatus = async (config: PublishConfig, project: IBotProject) => {
```

Comment:
what's the difference between getStatus and getProvisionStatus?

Reply:
getStatus is for the publishing - different return types.